lib/vtls/rustls.c

```diff
@@ -276,61 +277,114 @@ cr_send(struct Curl_easy *data, int sockindex,
   return plainwritten;
 }
 
+/* A server certificate verify callback for rustls that always returns
+   RUSTLS_RESULT_OK, or in other words disable certificate verification. */
+static enum rustls_result
+cr_verify_none(void *userdata UNUSED_PARAM,
+  const rustls_verify_server_cert_params *params UNUSED_PARAM)
+{
+  return RUSTLS_RESULT_OK;
+}
+
+static bool
+cr_hostname_is_ip(const char *hostname)
+{
+    struct in_addr in;
+#ifdef ENABLE_IPV6
+    struct in6_addr in6;
+    if(Curl_inet_pton(AF_INET6, hostname, &in6) > 0) {
+      return true;
+    }
+#endif /* ENABLE_IPV6 */
+    if(Curl_inet_pton(AF_INET, hostname, &in) > 0) {
+      return true;
+    }
+    return false;
+}
+
+static CURLcode
+cr_init_backend(struct Curl_easy *data, struct connectdata *conn,
+  struct ssl_backend_data *const backend) {
```

Comment:
per our code style that opening brace should be on the first column of the next line!

Reply:
Fixed, thanks!